content/guides/box-sign/sign-templates.md

```diff
@@ -0,0 +1,269 @@
+---
+rank: 2
+---
+
+# Create Sign Request with Sign Template
+
+The Sign Request API allows you use a predefined Box Sign
+template when creating a sign request.
+The template includes placeholders
+that are automatically populated with data
+when creating the request.
+
+## Create Template
+
+Start with creating a Box Sign template that
```

Comment:
Looks like the end of the sentence is missing?

Reply:
it is indeed. :D Fixed :)